dev/com.ibm.ws.classloading.bells/resources/com/ibm/ws/classloading/bells/internal/resources/BellConfigurationMessages.nlsprops

```diff
@@ -79,3 +79,21 @@ bell.error.ctor.useraction=If an exception occurred in the constructor, review a
 bell.no.services.config=CWWKL0058W: Cannot find the {0} service in META-INF/services folder from the {1} library.
 bell.no.services.config.explanation=There is no matching service declared in the META-INF/services folder.
 bell.no.services.config.useraction= No user action is required.
+
+# BETA
+# {0} - library identifier
+bell.spi.visibility.enabled=CWWKL0059I: BELL SPI visibility is enabled for the {0} library. The BELL will reference the {0} library through a unique class loader that can see SPI packages.
+bell.spi.visibility.enabled.explanation=To prevent applications and resources from seeing SPI packages, the BELL will reference the library through a unique class loader that is not shared with applications that reference the same library.
```

Comment:
```suggestion
bell.spi.visibility.enabled.explanation=To prevent applications and resources from seeing SPI packages, the BELL references the library through a unique class loader that is not shared with applications that reference the same library.
```